_opensearch/rest-api/create-alias.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Create alias
+parent: REST API reference
+grand_parent: OpenSearch
+nav_order: 8
+---
+
+# Create alias
+
+An alias is a virtual pointer that you can use to reference one or more indices. For example, if you have continuous streams of incoming data but only want to search through the past seven days at any time, you can use an alias to point to only indices created within the past seven days.
+
+## Example
+
+```json
+POST _aliases
+{
+  "actions": [
+    {
+      "add": {
+        "index": "movies",
+        "alias": "movies-alias1"
+      }
+    }
+  ]
+}
+```
+
```

Comment:
We want to include the "Path and HTTP methods" section on each of these REST API pages (see https://docs-beta.opensearch.org/opensearch/rest-api/cluster-allocation/#path-and-http-methods for example).

I missed flagging that that section was missing on the https://docs-beta.opensearch.org/opensearch/rest-api/create-index/ page, so I'd either add that section to this PR or do a separate commit -- should be an easy fix regardless.